### PR TITLE
 London Class 7 - Negin Dn - HTML/CSS - Week 2

### PR DESCRIPTION
**Pull request by NeginDn**
# Your Details

- Your Name: Negin Dn
- Your City:London
- Your Slack Name:Negin Dn

# Homework Details

- Module:HTML/CSS
- Week:2

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,16 +1,59 @@
...
+<title>Responsive Cake webpage</title>
+<link rel="stylesheet" href="./style.css"></head>
 </head>
-
 <body>
-    <!-- Add your markup here -->
+<!-- Add your markup here -->
+<header>
+<section class= "image-container" >
+<div class= "logo">
```
Comment (LucyMac) [inline index.html:10]: Watch your indentation please, it makes it harder to read without it.

File: index.html [hunk 2]
```diff
@@ -1,16 +1,59 @@
...
+    <img src="image/logo.jpg" alt="">
+</div>
+<div class= "textMenu-container" >
+    <img class="menu-icon" src="image/menu-hamburger.svg" alt=""/>
+    <section class="text">
+    <p>Best cake in town </p>
+    <p>Deliver to your door</p>
+</section>
+</div>
+</section>
```
Comment (LucyMac) [inline index.html:20]: You're closing 2 sections here but I only see one open on line 9 - indentation would help avoid this 😉

File: index.html [hunk 3]
```diff
@@ -1,16 +1,59 @@
...
+        <div>
+            <img class="intro-image" src="image/cake.jpg" alt="Cake picture">
+            
+        </div>
+    </section>
+  
+    <section id="product-container">
+        <div>
+            <div class="product-text"><p>Products</p></div>
+             <a href="#pastries"><img class= "product-image" src="image/pastries.jpg" alt="pastries"></a>
```
Comment (LucyMac) [inline index.html:43]: Nice touch making your images into links 👍

File: index.html [hunk 4]
```diff
@@ -1,16 +1,59 @@
...
+             <a href="#birthdayCakes"><img class= "product-image" src="image/BirthdayCake.jpg" alt="birthday cake"></a>
+             <a href="#cupcakes"><img class= "product-image" src="image/cupcake.jpg" alt="cupcake"></a>
+             <hr>
+        </div>
+    </section>
+</main>
 </body>
-
+<footer>
+    <div id="footer-container">
```
Comment (LucyMac) [inline index.html:53]: Do you need this div? It looks like the `footer` tag could fill this role as it wraps the same content as the div. Move your styles to the `footer` tag (and don't use IDs in your CSS!)

File: style.css [hunk 5]
```diff
@@ -1 +1,160 @@
...
+  background-color: rgb(179, 176, 176);
+}
+
+.footer-item {
+  text-decoration: none;
+  color: rgb(255, 226, 231);
+  font-size: 10px;
+}
+
+@media (min-width: 540px) {
```
Comment (LucyMac) [inline style.css:112]: Can you add another media query for tablet?

File: style.css [hunk 6]
```diff
@@ -1 +1,160 @@
...
+  .intro-text {
+    text-align: justify;
+    padding-right: 40px;
+  }
+
+  /*product*/
+
+  #Footer-container {
+    flex-direction: row;
+  }
```
Comment (LucyMac) [inline style.css:147]: Don't use IDs in your CSS. 
Only use IDs in your HTML for internal anchor links or for Javascript - but not in your CSS.

File: style.css [hunk 7]
```diff
@@ -1 +1,160 @@
...
+
+  .product-image {
+    width: 21%;
+    justify-content: space-between;
```
Comment (LucyMac) [inline style.css:151]: Using flexbox rules without `display: flex` has no effect. This rule will not do anything until you make `product-image` a flex container.

**Review by LucyMac — COMMENTED** [review]
Nice looking page Negin - add a little more CSS and media queries to make your page more responsive. 
Also watch out with flexbox - you need to use `display: flex` for the other flex rules to work.

File: index.html [hunk 8]
```diff
@@ -1,16 +1,59 @@
...
+    <img class="menu-icon" src="image/menu-hamburger.svg" alt=""/>
+    <section class="text">
+    <p>Best cake in town </p>
+    <p>Deliver to your door</p>
+</section>
+</div>
+</section>
+ <div class= "hamburger-menu">
+    <div><a class="hamburger-item" href="#Aboutus"><p>About us</p></a></div>
+    <div><a class="hamburger-item" href="#CustomerCare"><p>Customer Care</p></a></div>
```
Comment (louisechow) [inline index.html:23]: The `<p>` tag in your links is not necessary. You can simplify the link and just put
`<a class="hamburger-item" href="#Aboutus">About us</a>`

**Pull request closed without merging**